bundle/config/mutator/python/python_mutator.go

```diff
@@ -311,6 +319,21 @@ func createInitOverrideVisitor(ctx context.Context) merge.OverrideVisitor {
 	}
 }
 
+func isOmitemptyDelete(left dyn.Value) bool {
```

Comment:
`omitempty` is specific to the JSON serializer. Here we check if the incoming value is an empty collection.

Maybe `isEmptyCollection`?

Reply:
This was intentional because we want to reserve this method to specific behavior with omit empty and add a comment explaining that. For instance, we intentionally don't handle objects where all fields are empty.